modules/monitoring-setting-audit-log-levels-for-the-prometheus-adapter.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="setting-audit-log-levels-for-the-prometheus-adapter_{context}"]
+= Setting audit log levels for the Prometheus Adapter
+
+[role=_abstract]
+In default platform monitoring, you can configure the audit log level for the Prometheus Adapter.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+
+.Procedure
+
+You can set an audit log level for the Prometheus Adapter in the default `openshift-monitoring` project:
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add `profile:` under `audit:` for the `k8sPrometheusAdapter` component under `data/config.yaml`:
```

Comment:
"...under...under..." 
Reading this, I got started trying to figure out what went under what until I looked at the example.
Would `k8sPrometheusAdapter/audit/profile`-style notation work here?

Reply:
Change to `Add `profile:` in the `k8sPrometheusAdapter/audit` section under `data/config.yaml``